frontend/src/scenes/session-recordings/sessionRecordingLogic.ts

```diff
@@ -270,8 +272,15 @@ export const sessionRecordingLogic = kea<sessionRecordingLogicType>({
                 }
             },
             loadRecordingSnapshots: async ({ sessionRecordingId, url }, breakpoint): Promise<SessionPlayerData> => {
+                const snapshotParams = {}
+                if (values.featureFlags[FEATURE_FLAGS.TUNE_RECORDING_SNAPSHOT_LIMIT]) {
+                    snapshotParams['limit'] = 4
+                }
```

Comment:
nit: instead of creating a mutable dictionary at the top here can we simplify this to a one-liner that's more javascripty?

```
toParams({
    limit: ifFeatureFlagOn ? 4 : undefined
})
```

Reply:
Cool! I always thought that would add a blank query param to the URL (e.g. `?limit=`), but looks like it doesn't!